cinema/tests/test_movie_api.py

```diff
@@ -157,3 +154,12 @@
         res = self.client.get(MOVIE_SESSION_URL)
 
         self.assertIn("movie_image", res.data[0].keys())
+
+
+class UnauthenticatedMovieApiTests(TestCase):
+    def setUp(self):
+        self.client = APIClient()
+
+    def test_auth_required(self):
+        res = self.client.get(MOVIE_URL)
+        self.assertEqual(res.status_code, status.HTTP_401_UNAUTHORIZED)
```

Comment:
This test only checks if authentication is required to retrieve movies. It would be more comprehensive to also check if authentication is required for other operations such as creating, updating and deleting movies.